resources/views/components/badge.blade.php

```diff
@@ -7,13 +7,13 @@
 <span
     @class([
         'filament-badgeable-badge px-2 inline-flex  text-xs font-medium',
-        'text-primary-800 bg-primary-500/20 dark:text-primary-400' => $color === 'primary',
-        'text-secondary-800 bg-secondary-500/20 dark:text-secondary-400' => $color === 'secondary',
-        'text-success-800 bg-success-500/20 dark:text-success-400' => $color === 'success',
-        'text-warning-800 bg-warning-500/20 dark:text-warning-400' => $color === 'warning',
-        'text-danger-800 bg-danger-500/20 dark:text-danger-400' => $color === 'danger',
-        'text-gray-800 bg-gray-500/20 dark:text-gray-300' => $color === 'default' || $color === 'gray',
-        'text-gray-800' => $invertTextColor() && ! $getTextColor(),
+        'text-primary-700 bg-primary-500/10 dark:text-primary-500' => $color === 'primary',
+        'text-secondary-700 bg-secondary-500/10 dark:text-secondary-500' => $color === 'secondary',
+        'text-success-700 bg-success-500/10 dark:text-success-500' => $color === 'success',
+        'text-warning-700 bg-warning-500/10 dark:text-warning-500' => $color === 'warning',
+        'text-danger-700 bg-danger-500/10 dark:text-danger-500' => $color === 'danger',
+        'text-gray-700 bg-gray-500/10 dark:text-gray-500' => $color === 'default' || $color === 'gray',
```

Comment:
Can you do me a favor and refactor these to a match statement? Other than that looks good.

Reply:
Please see my changes.